native_client/java/libdeepspeech/src/androidTest/java/org/mozilla/deepspeech/libdeepspeech/test/BasicTest.java

```diff
@@ -149,4 +155,28 @@ public void loadDeepSpeech_sttWithMetadata_withLM() {
         assertEquals("she had your dark suit in greasy wash water all year", decoded);
         m.freeModel();
     }
+
+    @Test
+    public void loadDeepSpeech_HotWord() {
+        DeepSpeechModel m = new DeepSpeechModel(modelFile);
+        m.addHotWord( word1, boost1);
+
+        String decoded = doSTT(m, true);
+        assertEquals("she had your dark suit in greasy wash water all year", decoded);
+        m.freeModel();
```

Comment:
let's not do this, but rather use the API entry point to cleanup, `eraseHotWord` or `clearHotWords` ?

Reply:
ok I will `eraseHotWord` in each for loop iteration.